### PR TITLE
Documentation: improve README.

### PR DESCRIPTION
**Pull request by cellinacywinska**
Hi everyone! I was assigned to attempt restructuring the README and so I wanted to share my draft. I would really appreciate feedback about sections, level of detail, and everything else!

Additionally, I went ahead and added lists of contents across the siembol/docs/services folder.

Please take a look. Thank you!

### CODE REVIEW

**Review by mariannovotny — CHANGES_REQUESTED** [review]
Hi,

Thanks for your PR. In my opinion the compressed readme(the current state) is simpler and easier to navigate. It is no need to add additional words without adding new information. What could be improved:
- add more badges (license, all badges of BH at which that we presented Siembol)
- add links to license, code of conduct, changelog under Introduction
- create a page for Siembol services with links to related guides. Link the page from the readme
Can you also please better name PR? Something like Documentation: improve README.
Thanks,
Marian

**Comment by stackedsax** [comment]
Interesting. I honestly find the new layout much easier to understand and more welcoming.  The addition of the introduction section gives me much more of a sense of why this project exists and the new headings draw my eye towards what I'm interested in instead of having to parse a big list of hyperlinks.

I agree with these suggestions: 
> add more badges (license, all badges of BH at which that we presented Siembol)
> add links to license, code of conduct, changelog under Introduction

It looks like some of them may already have been added? 

A new page for the services could be good, too.  But perhaps that could be a subsequent PR?

**Comment by tabathad** [comment]
I find @cellinacywinska's updates improve the readability of the information presented with clear formatting. The updates provide hierarchy and allow for easy visual scanning to find the information a user is likely looking for when visiting the README. 

@mariannovotny suggestions seem to be incorporated and additionally improve the usability of the readme.

File: README.md [hunk 1]
```diff
@@ -1,37 +1,80 @@
 ![Siembol](logo.svg)
 
 [![Black Hat Arsenal](https://raw.githubusercontent.com/toolswatch/badges/54ad78bc63b24ce445e8241f179fe1ddeecf8eef/arsenal/usa/2021.svg)](https://www.blackhat.com/us-21/arsenal/schedule/index.html#siembol-an-open-source-real-time-siem-tool-based-on-big-data-technologies-24038)
```
Comment (mariannovotny) [inline README.md:3]: Can you please add all badges of Black Hats with siembol demo? +EU 2021, USA 2022?

File: README.md [hunk 2]
```diff
@@ -1,37 +1,80 @@
...
-    - [Setting up Kerberos for external dependencies](/docs/deployment/how-tos/how_to_set_up_kerberos_for_external_dependencies.md)
-    - [Customize Helm chart](/docs/deployment/how-tos/how_to_customize_helm_charts.md)
+Introduction
+------------
+
+Siembol is an open-source, real-time security information and event management tool developed in-house at G-Research.
+ 
+Siembol is designed to support the following:
+
+* to centralize both security data collecting and the monitoring of logs from different sources.
```
Comment (mariannovotny) [inline README.md:17]: Please reformulate or use existing wording from introduction or black hat abstract.

File: README.md [hunk 3]
```diff
@@ -1,37 +1,80 @@
...
+* as a tool for detecting attacks or leaks by teams responsible for the system platform.
+
+For a full description, visit: [Introduction](/docs/introduction/introduction.md).
```
Comment (mariannovotny) [inline README.md:20]: it is an introduction and not full description?

File: README.md [hunk 4]
```diff
@@ -1,37 +1,80 @@
...
+
+How to contribute
+-----------------
+
+If you wish to contribute to Siembol, first read: [Contribution Guide](/docs/introduction/how-tos/how_to_contribute.md).
+
+#### Code of Conduct
+G-Research has adopted a Code of Conduct that is to be honored by everyone who participates in the Siembol community formally or informally.
+Please read the full text: [Code of Conduct](/CODE_OF_CONDUCT.md)
+
```
Comment (mariannovotny) [inline README.md:35]: Please add link to changelog

File: README.md [hunk 5]
```diff
@@ -1,37 +1,80 @@
...
+- [Setting up OAUTH2 OIDC](/docs/siembol_ui/how-tos/how_to_setup_oauth2_oidc_in_siembol_ui.md)
+- [Modifying the layout](/docs/siembol_ui/how-tos/how_to_modify_ui_layout.md)
+- [Managing applications](/docs/siembol_ui/how-tos/how_to_manage_applications.md)
+- [Use ui-bootstrap file](/docs/siembol_ui/how-tos/how_to_use_ui_bootstrap_file.md)
+- [Filter configs and save searches](/docs/siembol_ui/how-tos/how_to_filter_configs_and_save_searches.md)
+
+Services
+---------
+
+To explore Siembol's services, visit: [Siembol services](/docs/services).
```
Comment (mariannovotny) [inline README.md:58]: can you please prepare very simple page for services - just placeholder for links to underlaying documentation

**Review by mariannovotny — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (cellinacywinska) [reply README.md:35]: done!

File: README.md (hunk 3 above)
Comment (cellinacywinska) [reply README.md:20]: just changed the wording!

File: README.md (hunk 2 above)
Comment (cellinacywinska) [reply README.md:17]: done.

File: README.md (hunk 5 above)
Comment (cellinacywinska) [reply README.md:69]: Could you please confirm if my understanding is correct?
I should create a separate file with all the links that are currently listed in the readme. Then, I should link that file in the readme under 'Services' section and remove the rest.

Comment (cellinacywinska) [reply README.md:69]: Thank you, I think I get it! Does /docs/services seem appropriate for the location of the new file? Also does the name "links_to_services" work?

Comment (cellinacywinska) [reply README.md:69]: got it! thank you

File: README.md [hunk 6]
```diff
@@ -1,37 +1,81 @@
...
+- [Modifying the layout](/docs/siembol_ui/how-tos/how_to_modify_ui_layout.md)
+- [Managing applications](/docs/siembol_ui/how-tos/how_to_manage_applications.md)
+- [Use ui-bootstrap file](/docs/siembol_ui/how-tos/how_to_use_ui_bootstrap_file.md)
+- [Filter configs and save searches](/docs/siembol_ui/how-tos/how_to_filter_configs_and_save_searches.md)
+
+Services
+---------
+
+To explore Siembol's services, visit: [Siembol services](/docs/services/services.md).
+
```
Comment (mariannovotny) [inline README.md:70]: Can you please add list of guides similar to other parts? e.g. There you will find guides on:

File: docs/services/services.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# Siembol's services:
```
Comment (mariannovotny) [inline docs/services/services.md:1]: 'Siembol services' to be consistent?

**Review by mariannovotny — CHANGES_REQUESTED** [review]

File: docs/services/services.md (hunk 7 above)
Comment (cellinacywinska) [reply docs/services/services.md]: done!

File: README.md (hunk 6 above)
Comment (cellinacywinska) [reply README.md:80]: There is an issue with my commit "77adc63"- two checks didnt pass for some reason. I'm not sure if that's okay.

Comment (mariannovotny) [reply README.md:80]: The tests can sometimes fail for example the server is busy. Not related to your changes.

File: docs/services/services.md (hunk 7 above)
Comment (mariannovotny) [reply docs/services/services.md]: Perfect! Can you please remove ':' ? Then we can merge it.

Comment (cellinacywinska) [reply docs/services/services.md]: done!

**Review by mariannovotny — APPROVED** [review]
LGTM! Thanks for PR.

**Pull request merged**